xml/en/community.xml

```diff
@@ -5,18 +5,89 @@
 
 <!DOCTYPE article SYSTEM "../../dtd/article.dtd">
 
-<article name="nginx: support"
-         link="/en/support.html"
+<article name="nginx: community"
+         link="/en/community.html"
          lang="en"
-         rev="10">
+         rev="11">
```

Comment:
I'd put version 1 instead of 11 (it's a new page and we might want to revive support.xml with better content at some time)